repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -166,6 +166,14 @@ object ProjectFileContent {
   implicit val projectFileContentFormat: Format[ProjectFileContent] = Json.valueFormat[ProjectFileContent]
 }
 
+final case class GitLabFileContent(content: String)
+
+object GitLabFileContent {
+  implicit val encodingProjectFileContentFormat: Reads[GitLabFileContent] = {
+    (JsPath \ "content").read[String].map(GitLabFileContent.apply)
```

Comment:
```suggestion
  implicit lazy val projectFileContentFormat: OFormat[GitLabFileContent] = Json.format[GitLabFileContent]
```
should work as well I think